x-pack/legacy/plugins/maps/server/routes.js

```diff
@@ -414,6 +415,28 @@ export function initRoutes(server, licenseUid) {
     },
   });
 
+  server.route({
+    method: 'GET',
+    path: `${ROOT}/indexSettings`,
+    handler: async (request, h) => {
+      const { server, query } = request;
+
+      if (!query.indexPatternTitle) {
+        return h.response().code(400);
+      }
+
+      const { callWithRequest } = server.plugins.elasticsearch.getCluster('data');
+      try {
+        const resp = await callWithRequest(request, 'indices.getSettings', {
+          index: query.indexPatternTitle,
+        });
+        return getIndexPatternSettings(resp);
+      } catch (error) {
```

Comment:
log error. 400-error code is less debuggable (requires knowledge of the endpoints).